src/audioPlayer.js

```diff
@@ -0,0 +1,96 @@
+// Instantiate the PLAYER variable in the Webflow page's head since it is needed across multiple scripts.
+PLAYER = document.querySelector('[data-element=audio-player]')
+PLAYER.querySelector('source').src = '0'
+
+const AUDIO_TOGGLES = document.querySelectorAll('[data-element=player-toggle]')
+
+AUDIO_TOGGLES.forEach((podcastTrigger) => {
+    podcastTrigger.addEventListener('click', setPlayer.bind(podcastTrigger))
+})
+
+// Helper to safely call a function declared in the analytics script that should be loaded.
+function getAnalyticsEventProperties(eventName, dataset, podcastTrigger) {
+    if (typeof getEventProperties !== 'undefined') {
+        // eslint-disable-next-line no-undef
+        return getEventProperties(eventName, dataset, podcastTrigger)
+    } else {
+        console.warn('getEventProperties function not loaded!')
+        return {}
+    }
+}
+
+// Attempt to retrieve animation info, if it exists, and sync the audio player to the animation.
+function syncAudioPlayerAndAnimation() {
+    if (typeof CURRENT_ANIMATION_INFO !== 'undefined' && typeof ANIMATIONS !== 'undefined') {
+        // eslint-disable-next-line no-undef
+        if (CURRENT_ANIMATION_INFO.name in ANIMATIONS) {
+            // eslint-disable-next-line no-undef
+            const animation = ANIMATIONS[CURRENT_ANIMATION_INFO.name]
+            if (PLAYER.querySelector('source').src === animation.expectedAudioSrc) {
+                PLAYER.currentTime = ((
+                    // eslint-disable-next-line no-undef
+                    (new Date()).valueOf() - CURRENT_ANIMATION_INFO.timeScrolledIntoView
+                ) % animation.duration) / 1000
+            }
+        }
+    } else {
+        console.warn('There is no animation loaded on this page.')
+    }
+}
+
+async function setPlayer() {
```

Comment:
A docstring with a sample `html` element required to display a functioning audio player would be useful